gcc/rust/resolve/rust-forever-stack.h

```diff
@@ -521,6 +521,12 @@ template <Namespace N> class ForeverStack
 
   std::string as_debug_string ();
 
+  /**
+   * Used to check if a module is a decendent of another module
+   * Intended for use in the privacy checker
+   */
+  bool is_module_decendent (NodeId parent, NodeId child) const;
```

Comment:
typo
```suggestion
  bool is_module_descendant (NodeId parent, NodeId child) const;
```

but also, I'd be happy with something like `is_module_child` which seems a bit simpler

Reply:
I thought I had it spelled right, but it looks like I [didn't](https://www.merriam-webster.com/dictionary/decedent)

Reply:
Should be fixed now